index.js

```diff
@@ -1943,6 +1943,29 @@
         return either.isRight ? [either.value] : [];
       }));
 
+  //# toEither :: a -> b? -> Either a b
+  //.
+  //. Takes a failure value, 'x', and a possibly null or undefined value, 'y'.
+  //. Returns a Right of 'y' if it is defined and not null, otherwise returns
+  //. a Left of 'x'.
+  //.
+  //. See also ['encaseEither'](#encaseEither)
+  //.
+  //. ```javascript
+  //. > S.toEither("No string", "ftp://example.com/")
+  //. Right("ftp://example.com/")
+  //.
+  //. > S.toEither("Invalid protocol", "ftp://example.com/".match(/^https?:\/\//))
+  //. Left("Invalid protocol")
+  //. ```
+  S.toEither =
+  def('toEither',
+      {},
+      [a, b, $Either(a, b)],
+      function(x, y) {
+        return y == null ? Left(x) : Right(y);
+      });
```

Comment:
Let's define the anonymous function on one line: `function(x, y) { return y == null ? Left(x) : Right(y); }`.